Source/Core/DiscIO/NANDContentLoader.cpp

```diff
@@ -495,6 +487,7 @@ u64 CNANDContentManager::Install_WiiWAD(const std::string& filename)
 
   tmd_file.WriteBytes(content_loader.GetTMDHeader(), CNANDContentLoader::TMD_HEADER_SIZE);
 
+  CSharedContent shared_content{Common::FromWhichRoot::FROM_CONFIGURED_ROOT};
```

Comment:
Could you make it so that this object only gets constructed if we need to use it? It would save a bit of file I/O.